packages/web-components/src/radio-group/README.md

```diff
@@ -0,0 +1,127 @@
+# Radio Group
+
+> RadioGroup lets people select a single option from two or more Radio items. Use RadioGroup to present all available choices if there's enough space. For more than 5 choices, consider using a different component such as Dropdown.
+
+<br />
+
+## **Design Spec**
+
+[Link to Radio Design Spec in Figma](https://www.figma.com/file/4XWsJrlpEcuEpUnZbtoIBU/Radio?node-id=1295%3A0&t=2wXnjT4ybxIxT6wu-0)
+
+<br />
+
+## **Engineering Spec**
+
+A radio group is a set of checkable buttons, known as radio buttons, where no more than one of the buttons can be checked at a time. Some implementations may initialize the set with all buttons in the unchecked state in order to force the user to check one of the buttons before moving past a certain point in the workflow.
```

Comment:
Let's denote that this is a quote from the W3C, without that, it's written in a way where I might mistake that these should be built _with_ buttons, which is inaccurate as the roles are different.